src/app/file-manager/actions/downloadables.actions.ts

```diff
@@ -66,6 +68,19 @@ export class SelectItem implements Action {
     constructor(public payload: any) { }
 }
 
+export class SetDestination implements Action {
+    readonly type = SET_DESTINATION;
+
+    constructor(public payload: any) { }
```

Comment:
is it possible to use a more narrow type here, e.g. `string` for this payload and `boolean` for the other?